### DISCUSSION TITLE
Enabling react compiler in tests?

### DISCUSSION THREAD

**Discussion by yepitschunked** [reactwg/react-compiler#45] [General Questions]
Are there any recommendations for/against enabling React Compiler in test environments? I can see arguments both ways - React Compiler should be a transparent improvement, so it should not be a factor in tests. Code should not *depend* on the compiler in order to work properly. On the flip side, enabling the compiler in tests could reveal some edge cases (e.g, the compiler not bailing out when it should).

**Comment by poteto** [comment]
For library authors we do recommend testing with the compiler on/off. For apps having this setup can be helpful as regression tests especially when you're rolling out a new compiler version.
